api/models.py

```diff
@@ -3,8 +3,8 @@
 
 class Tab(db.Model):
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String, nullable=False)
-    description = db.Column(db.String)
+    name = db.Column(db.String(1000), nullable=False)
+    description = db.Column(db.String(1000))
```

Comment:
Don't use such huge strings, they are converted to `VARCHAR`, better use `TEXT` if you expect there to be a rather lengthy string. For `name`, `key` und `column_type` I personally would start with `CHAR` or `VARCHAR(255)`.

If you are unfamiliar with those specific datatypes, you may want to take a look at the [documentation](https://dev.mysql.com/doc/refman/8.0/en/char.html).

Reply:
I know what the different datatypes mean.
Just put a random bit number there but we can use 255 which is probably gonna be enough since the longes one currently is like 150 currently if I'm correct.